java/arcs/core/storage/BackingStore.kt

```diff
@@ -26,60 +25,47 @@ import kotlinx.coroutines.sync.withLock
 import kotlinx.coroutines.withContext
 
 /**
- * An [ActiveStore] that allows multiple CRDT models to be stored as sub-keys of a single
- * storageKey location.
+ * An collection of [DirectStore]s that allows multiple CRDT models to be stored as sub-keys
+ * of a single storageKey location.
  *
  * This is what *backs* Entities.
  */
 class BackingStore<Data : CrdtData, Op : CrdtOperation, T>(
-    private val options: StoreOptions<Data, Op, T>
-) : ActiveStore<Data, Op, T>(options) {
+    val storageKey: StorageKey,
+    val backingType: Type,
+    val callbackFactory: ((String) -> ProxyCallback<Data, Op, T>)
+) {
     private val storeMutex = Mutex()
     /* internal */ val stores = mutableMapOf<String, StoreRecord<Data, Op, T>>()
-    private val callbacks = RandomProxyCallbackManager<Data, Op, T>(
-        "backing",
-        Random
-    )
-
-    @Deprecated(
-        "Use getLocalData(muxId) instead",
-        replaceWith = ReplaceWith("getLocalData(muxId)")
-    )
-    override suspend fun getLocalData(): Data =
-        throw UnsupportedOperationException("Use getLocalData(muxId) instead.")
 
     /**
-     * Gets data from the store corresponding to the given [muxId].
+     * Gets data from the store corresponding to the given `referenceId`.
```

Comment:
I believe that in KDoc you can refer to argument names using square brackets: `[referenceId]`. (At least, IntelliJ allows it)

Reply:
Here and throughout file